src/PrestaShopBundle/Resources/config/routing/admin/improve/international/zone.yml

```diff
@@ -0,0 +1,77 @@
+admin_zones_index:
+  path: /
+  methods: GET
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Improve\International\Zone:index'
+    _legacy_controller: AdminZones
+    _legacy_link: AdminZones
+
+admin_zones_search:
+  path: /
```

Comment:
```suggestion
  path: /search
```

Reply:
Why? All others have this as route `/` as POST.

Reply:
Why did they do this :scream: